CMakeLists.txt

```diff
@@ -255,6 +255,7 @@ option(FLB_OUT_PROMETHEUS_REMOTE_WRITE "Enable Prometheus remote write plugin"
 option(FLB_OUT_S3                      "Enable AWS S3 output plugin"                  Yes)
 option(FLB_OUT_VIVO_EXPORTER           "Enabel Vivo exporter output plugin"           Yes)
 option(FLB_OUT_WEBSOCKET               "Enable Websocket output plugin"               Yes)
+option(FLB_OUT_OCI_LOGAN               "Enable OCI Logging analytics plugin"          Yes)
```

Comment:
Per the docs PR, I would expand OCI to be Oracle Cloud Infrastructure as it is a fairly generic acronym and may be confused with open container initiative.

Reply:
what about `oracle_log_analytics` ? azure has `azure_logs ingestion`

Reply:
Yeah sounds good to me, making it clear it is Oracle I think is the key one - we have AWS/Google equivalents too.

Reply:
Good idea, I'll make these changes

Reply:
Refactored @edsiper